index.bs

```diff
@@ -918,6 +924,20 @@ An aggregatable dedup key is a [=struct=] with the following items:
 
 </dl>
 
+<h3 dfn-type=dfn>Aggregatable bucket</h3>
+
+An aggregatable bucket is a [=struct=] with the following items:
+
+<dl dfn-for="aggregatable bucket">
+: <dfn>bucket</dfn>
+:: A [=string=].
```

Comment:
Should this be nullable?

Just thinking whether there's a use case where for example we want the following behavior:
filter 1: no bucket
filter 2: bucket 1
filter 3: no bucket

Reply:
Good point, I think that's a valid case to explicitly select no bucket with filters.

Reply:
Done.